src/modules/Update.rb

```diff
@@ -850,6 +859,11 @@ def restore_backup
 
   private
 
+    # @return [Integer] boot time found in /proc/stat
+    def boot_time
+      @boot_time ||= IO.read("/proc/stat")[/btime.*/].split.last.to_i
```

Comment:
It would be nice to add a test for parsing the file, the added test mocks the method completely so it's skipped.